### PR TITLE
Fixes for issues #9, #11, #12

### PR DESCRIPTION
**Pull request by chemichelle**

### CODE REVIEW

**Comment by ahmetb** [comment]
please `git rm .DS_Store`.

**Comment by chemichelle** [comment]
Done ✅

File: index.html [hunk 1]
```diff
@@ -182,12 +204,18 @@ <h2>Congrats, you successfully deployed a container image to Cloud Run</h2>
...
 		<div class="details">
-			<p>This created the revision <span class="code">{{.Revision}}</span> of the Cloud Run service <span
+		<p>This created the revision <span class="code">{{.Revision}}</span> of the Cloud Run service <span
 					class="code">{{.Service}}</span>{{if .ProjectFound}} in the GCP project <span
 					class="code">{{.Project}}</span>{{end}}.</p>
-
-			<p class="callout">
-				<label><img src="/assets/lightbulb_icon.svg" style="margin:0px 8px 0px 0px; vertical-align:text-top;">
+					
+		<!-- Hard coded for testing -->
```
Comment (steren) [inline index.html]: Remove

**Comment by steren** [comment]
This PR does many things at once. It would have been preferred to send smaller PRs, each addressing one of the fixed issues. This makes things easier to review and is less error prone. That being said, let's do all in one.

I'll review.

File: assets/cloud_bg.svg [hunk 2]
```diff
@@ -1 +1,38 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="1444px" height="679px" viewBox="0 0 1444 679" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" jetway-hooks="{}">
+    <!-- Generator: Sketch 62 (91390) - https://sketch.com -->
```
Comment (steren) [inline assets/cloud_bg.svg]: Please minify this file, like it was originally.

File: assets/dark_bg.svg [hunk 3]
```diff
@@ -0,0 +1 @@
....498 0 008.185 15.09c.435.185.883.339 1.34.461 1.21.354 2.465.532 3.727.529.6.002 1.2-.04 1.796-.124.409-.05.815-.123 1.216-.22 6.696-1.537 11.17-7.85 10.4-14.67-.77-6.821-6.542-11.978-13.412-11.986zm5.93 17.883c.046.128.087.255.121.388.224.87-.069 1.797-.379 2.773l-.109.355a5.448 5.448 0 00-.281 1.492 11.56 11.56 0 01-3.34 1.109 14.242 14.242 0 00-.65-1.287 4.817 4.817 0 00-2.473-2.218 5.759 5.759 0 00-2.067-.216 3.302 3.302 0 00-2.09.654 2.278 2.278 0 00-.734 1.586c-2.73-1.63-4.628-4.293-5.228-7.34-.6-3.046.151-6.196 2.071-8.681.08.132.137.276.166.427 0 .464-.14.919-.401 1.308-.154.28-.287.572-.396.871a4.267 4.267 0 00.246 3.095c.432.874.972 1.696 1.608 2.445a9.878 9.878 0 002.296 2.34 5.41 5.41 0 002.933.832c1.41.012 2.77-.51 3.788-1.453 1.734-1.757 2.009-4.43.666-6.482a9.758 9.758 0 00-1.573-1.664c-.246-.228-.499-.455-.734-.693-.833-.843-1.229-1.664-1.148-2.374a4.225 4.225 0 011.148-2.107c.27-.333.519-.681.746-1.043 3.059.04 5.973 1.263 8.093 3.394a4.943 4.943 0 00-.878 2.773 7.64 7.64 0 00.626 2.529c.12.321.24.643.338.965.138.443.511 1.94-.327 2.723-.23.196-.483.367-.752.51-.695.29-1.23.853-1.47 1.547-.078.488-.014.987.185 1.442z" fill="#6C6D6E" fill-rule="nonzero"/></g></svg>
```
Comment (steren) [inline assets/dark_bg.svg]: Is this file empty? If so, why do you need a file at all?

File: index.html [hunk 4]
```diff
@@ -30,7 +30,7 @@
  h2 {
 	font-size: 18px;
 	font-weight: normal;
-	margin-top: 4px;
+	margin-top: 0px;
```
Comment (steren) [inline index.html]: Remove. 0px is the default

File: index.html [hunk 5]
```diff
@@ -145,12 +145,31 @@
 	.container {
 	  margin-top: 0;
 	}
+  }
 
-	.hero {
-	  margin-top: 0;
-	  margin-bottom: 24px;
+@media (prefers-color-scheme: light) {
```
Comment (steren) [inline index.html]: It's preferred to have this as a default (no need for the @media).

File: index.html [hunk 6]
```diff
@@ -145,12 +145,31 @@
...
+		background-position: center;
+		background-attachment: fixed;
+		line-height: 1.6;
 	}
-  }
+}	
+
+@media (prefers-color-scheme: dark) {
+	body {
+		background: black url('/assets/dark_bg.svg');
```
Comment (steren) [inline index.html]: Only add items that change in the @media block, the rest will use the same.values you set in the default.

**Comment by steren** [comment]
Please also edit the title of the PR to summarize what your changes are doing.

Only use the "fix #11..." in the description or at the end of the title

File: assets/dark_bg.svg (hunk 3 above)
Comment (steren) [reply assets/dark_bg.svg]: Sorry, it's not empty, the app had a problem

File: assets/cloud_bg.svg (hunk 2 above)
Comment (steren) [reply assets/cloud_bg.svg]: Also, why is it changing ? Maybe you overrode it?

File: index.html [hunk 7]
```diff
@@ -182,12 +204,18 @@ <h2>Congrats, you successfully deployed a container image to Cloud Run</h2>
 		{{end}}
 
 		<div class="details">
-			<p>This created the revision <span class="code">{{.Revision}}</span> of the Cloud Run service <span
+		<p>This created the revision <span class="code">{{.Revision}}</span> of the Cloud Run service <span
```
Comment (steren) [inline index.html]: Revert your change for this line. The indentation you are using is not correct

File: index.html [hunk 8]
```diff
@@ -182,12 +204,18 @@ <h2>Congrats, you successfully deployed a container image to Cloud Run</h2>
...
-			<p class="callout">
-				<label><img src="/assets/lightbulb_icon.svg" style="margin:0px 8px 0px 0px; vertical-align:text-top;">
+					
+		<!-- Hard coded for testing -->
+		<!--	<p>This created the revision <span class="code">hello-celebration-00001-yoc</span> of the Cloud Run service <span
+						class="code">hello-celebration</span> in the GCP project <span
+						class="code">serverless-ux-playground</span></p> -->
+	
+
+		<p class="callout">
```
Comment (steren) [inline index.html]: Also revert. The indentation is incorrect and you are not adding anything here as far as I can see.

File: index.html [hunk 9]
```diff
@@ -159,7 +178,10 @@
 
 		<div class="hero">
 			<div style="text-align:center;">	
-				<img src="/assets/celebration.svg">
+				<picture>
+					<source srcset="/assets/celebration-dark.svg" alt="celebration" media="(prefers-color-scheme: dark)">
+					<img src="/assets/celebration.svg" alt="celebration" width="427" height="231">
```
Comment (steren) [inline index.html]: The `alt` attribute should be a human readable description of the image. Please provide more details than "celebration"

File: index.html [hunk 10]
```diff
@@ -182,12 +204,18 @@ <h2>Congrats, you successfully deployed a container image to Cloud Run</h2>
...
-				<label><img src="/assets/lightbulb_icon.svg" style="margin:0px 8px 0px 0px; vertical-align:text-top;">
+					
+		<!-- Hard coded for testing -->
+		<!--	<p>This created the revision <span class="code">hello-celebration-00001-yoc</span> of the Cloud Run service <span
+						class="code">hello-celebration</span> in the GCP project <span
+						class="code">serverless-ux-playground</span></p> -->
+	
+
+		<p class="callout">
+				<label><img src="/assets/lightbulb_icon.svg" alt="lightbulb_icon" style="margin:0px 8px 0px 0px; vertical-align:text-top;">
```
Comment (steren) [inline index.html]: Also use a more descriptive `alt`. Expect a human visually impaired to read it and understand

File: index.html (hunk 4 above)
Comment (steren) [reply index.html]: (I think, please confirm)

**Comment by steren** [comment]
You will also need to merge the latest changes from master

**Comment by steren** [comment]
I suggest to close this PR and do smaller ones for each issue.

In addition to a cleaner commit history, PRs will be easier to review, which is less error prone.

File: index.html (hunk 4 above)
Comment (chemichelle) [reply index.html]: Seems like 0px it is not the default, so I'm keeping this line for now.

**Pull request closed without merging**